src/lib/components/GroupTree/components/Plot/group_tree.js

```diff
@@ -0,0 +1,464 @@
+import * as d3 from "d3";
```

Comment:
As you've mentioned, it would be good to have this ported to typescript and use WebGL. Not for this review, though..